tests/fread/test_xls.py

```diff
@@ -88,3 +90,113 @@ def test__excel_coords_to_range2d(seed):
         range2d = (row0, row1, col0, col1)
         excel_coords = _range2d_to_excel_coords(range2d)
         assert _excel_coords_to_range2d(excel_coords) == range2d
+
+
+def test_10k_diabetes_xlsx():
+    filename = find_file("h2o-3", "fread", "10k_diabetes.xlsx")
+    DT = dt.fread(filename)
+    assert DT.shape == (10000, 51)
+    assert DT.names[:4] == ("race", "gender", "age", "weight")
+    assert DT["readmitted"].stype == dt.bool8
+    assert DT[:, "num_lab_procedures":"number_inpatient"].stype == dt.int32
+    assert dt.unique(DT["gender"]).nrows == 2
+
+
+def test_large_ids_xlsx():
+    filename = find_file("h2o-3", "fread", "large_ids.xlsx")
+    DT = dt.fread(filename)
+    assert DT.shape == (53493, 3)
+    assert DT.stypes == (dt.int32, dt.int32, dt.int32)
+    assert DT.names == ("C1", "member_id", "loan_amnt")
+    assert DT.sum().to_tuples() == [(14436256829, 810493167234, 721084925)]
+
+
+def test_set_xls_new_xlsx():
+    filename = find_file("h2o-3", "fread", "test_set_xls_new.xlsx")
+    DT = dt.fread(filename)
+    assert DT.shape == (7, 1)
+    assert DT.names == ("Data",)
+    assert DT.to_list()[0] == ['2019-05-06 00:00:00',
+                               '2019-05-07 00:00:00',
+                               '2019-05-08 00:00:00',
+                               '2019-05-09 00:00:00',
+                               '2019-05-10 00:00:00',
+                               '2019-05-11 00:00:00',
+                               '2019-05-12 00:00:00']
```

Comment:
What about also adding `assert DT.stype == dt.str32` to check the stype explicitly?

Reply:
Eventually we will be creating a `date64` column here, so the test will have to be updated anyways.